src/Tools/CodeGenerator/Featurizers.yaml

```diff
@@ -2,6 +2,10 @@ featurizers:
   # ----------------------------------------------------------------------
   - name: CatImputerFeaturizer
     estimator_name: CatImputerEstimator
+
+    release_version: 0.3.0
+    deprecated_version: 0.3.6

```

Comment:
Are these flipped? If I am reading right it seems the deprecated_version is newer then the release_version.

Reply:
That's correct. deprecated_version is an indication that the featurizer should no longer be used (which implies that it has already been released). Let me know if you can think of a less confusing name.